src/Symfony/Component/Security/Http/Authenticator/AbstractLoginFormAuthenticator.php

```diff
@@ -32,6 +32,18 @@ abstract class AbstractLoginFormAuthenticator extends AbstractAuthenticator impl
      */
     abstract protected function getLoginUrl(Request $request): string;
 
+    /**
+     * Override to change the request conditions that have to be
+     * matched in order to handle the login form submit.
+     *
+     * This default implementation handles all POST requests to the
+     * login path (@see getLoginUrl()).
+     */
+    public function supports(Request $request): bool
+    {
+        return $this->getLoginUrl($request) === $request->getPathInfo() && $request->isMethod('POST');
```

Comment:
```suggestion
        return $request->isMethod('POST') && $this->getLoginUrl($request) === $request->getPathInfo();
```

micro optim, in case `getLoginUrl()` implem is "heavy"